public/javascripts/releases-controllers.js

```diff
@@ -241,6 +241,14 @@
             if ('undefined' === typeof(ticket.diffs)) {
                 ticket.diffs = [];
             }
+
+            // Remove empty requirements
+            for (var key in $scope.newDiff.requirements) {
+                if (!$scope.newDiff.requirements[key]) {
```

Comment:
When iterating over a hash, you should use the hasOwnProperty method to ensure that the key is not an inherited property.
Run jshint on this if in doubt